ru/turing/courses/lesson4/Main.java

```diff
@@ -3,21 +3,29 @@
 public class Main {
     public static void main(String[] args) {
 
-        //todo реализовать с учетом выбросов эксепшнов следующие кейсы
+        /*https://www.youtube.com/watch?v=JKLrn_0XNOU*/
 
-        // позитивные сценарии
         Gender maleGender = Gender.ofName("MALE");
+        System.out.println(maleGender);
         Gender femaleGender = Gender.ofName("FEMALE");
+        System.out.println(femaleGender);
 
         Gender maleLowerCaseCharacter = Gender.ofName("male");
+        System.out.println(maleLowerCaseCharacter);
         Gender femaleLowerCaseCharacter = Gender.ofName("female");
+        System.out.println(femaleLowerCaseCharacter);
 
         Gender maleGenderByChar = Gender.ofShortNameRepr('M');
+        System.out.println(maleGenderByChar);
         Gender femaleGenderByChar = Gender.ofShortNameRepr('F');
+        System.out.println(femaleGenderByChar);
 
-        // негативные сценарии
-        Gender nullGender = Gender.ofName(null);
-        Gender falseNameGender = Gender.ofName("Attack helicopter");
-        Gender falseCharReprName = Gender.ofShortNameRepr('\n');
+       try {
+           Gender nullGender = Gender.ofName(null);
+           Gender falseNameGender = Gender.ofName("Attack helicopter");
+           Gender falseCharReprName = Gender.ofShortNameRepr('\n');
+       }catch (Exception e){
```

Comment:
```suggestion
       }catch (GenderDoesNotExist e){
```
Можно, конечно, общий Exception ловить, но зачем тогда свой прописывали?